packages/perseus/src/__tests__/__snapshots__/server-item-renderer.test.tsx.snap

```diff
@@ -82,13 +82,21 @@ exports[`server item renderer should snapshot: initial render 1`] = `
                           <div
                             class="paragraph"
                             data-perseus-paragraph-index="0"
+                          >
+                            <div
+                              class="paragraph"
+                            >
+                              <strong>
+                                Your answer should be
+                              </strong>
+                               
+                            </div>
+                          </div>
+                          <div
+                            class="paragraph"
+                            data-perseus-paragraph-index="1"
```

Comment:
What is this `data-perseus-paragraph-index` attribute? Is it like our version of `tabIndex`?

Reply:
Question has been posted to the team.